### DISCUSSION TITLE
Multiple Temperature Sensor control

### DISCUSSION THREAD

**Discussion by Trebuin** [crystalidea/macs-fan-control#583] [Ideas]
I would pay for this to have the ability to control the fans using more than one sensor.  Currently, If I select GPU to keep the temp down at 80, I have managed to get the CPU above 110.  Vice versa, I have selected CPU to keep at 80 and gotten the GPU up to 110.  Both cases are really bad and makes this program unusable for me.  I would like both the CPU and GPU to be monitored for temp limits and kick the fan up when it exceeds 80.

**Comment by kevinlcahill-commits** [comment]
http://drfan.forbootcamp.org/index.htm
This software does what you want in bootcamp, sorry its 3 years too late
